app/services/roles_service.rb

```diff
@@ -77,9 +78,11 @@ def create_default_roles!
 
     def create_default_hyrax_groups_with_roles!
       # Prevent Hyrax::Groups from being created in the public schema
-      return '`AccountElevator.switch!` into an Account before creating default Hyrax::Groups' if Site.instance.is_a?(NilSite)
+      return '`AccountElevator.switch!` into an Account before creating default Hyrax::Groups' if Site.instance
+                                                                                                      .is_a?(NilSite)
 
-      default_hyrax_groups_with_roles = DEFAULT_HYRAX_GROUPS_WITH_ATTRIBUTES.deep_merge(DEFAULT_ROLES_FOR_DEFAULT_HYRAX_GROUPS)
+      default_hyrax_groups_with_roles = DEFAULT_HYRAX_GROUPS_WITH_ATTRIBUTES
+                                        .deep_merge(DEFAULT_ROLES_FOR_DEFAULT_HYRAX_GROUPS)
```

Comment:
This works, but I think there might be a more ideal breaking point. This might still be too long, but we can try it: 

```suggestion
      default_hyrax_groups_with_roles =
              DEFAULT_HYRAX_GROUPS_WITH_ATTRIBUTES.deep_merge(DEFAULT_ROLES_FOR_DEFAULT_HYRAX_GROUPS)
```